op-supervisor/supervisor/backend/syncnode/reset.go

```diff
@@ -79,11 +79,20 @@ func (t *resetTracker) bisectToTarget() {
 	internalCtx, iCancel := context.WithTimeout(t.managed.ctx, internalTimeout)
 	defer iCancel()
 
+	// TODO(#16026): this `if` is an artifact of trying to re-target bisection while the resetTracker is already active.
+	// We can simplify by targeting one thing at a time.
+
 	// initialize the start of the range if it is empty
 	if t.a == (eth.BlockID{}) {
 		t.managed.log.Debug("Start of range is empty, fetching the anchor block as starting point")
+		// TODO(#16026): change name: GetActivationBlock, Anchor point = get first block of DB
 		anchor, err := t.managed.backend.AnchorPoint(internalCtx, t.managed.chainID)
 		if err != nil {
+			if errors.Is(err, types.ErrFuture) {
+				// TODO(#16026): need to trigger pre-interop reset
+				t.managed.log.Info("Missing activation block", "err", err)
+				return
+			}
```

Comment:
```suggestion
		if errors.Is(err, types.ErrFuture) {
			// TODO(#16026): need to trigger pre-interop reset
			t.managed.log.Info("Missing activation block", "err", err)
			return
		} else if err != nil {
```
avoids one indentation

Reply:
I'm not a fan of checking the error-type as an alternative case of the err!=nil case.